dlrs/clearlinux/tensorflow/mkl/README.md

```diff
@@ -1,27 +1,173 @@
+
+Unpack the file into your chosen working directory. For example, if the tar file is in your Downloads folder and you have navigated to the directory you want to extract it into, run:
+```
+tar -xvf ~/Downloads/ssd_inception_v2_coco_2018_01_28.tar.gz 
+```
+
+### Clone the Model Optimizer
+
+Next we need the model optimizer directory, named dldt, found [here.](https://github.com/opencv/dldt) This guide will assume the parent directory is on the same level as the model directory, ie:
+```
+.
++--Working_Directory
+   +-- ssd_inception_v2_coco_2018_01_28
+   +-- dldt
+```
+
+From inside the working directory:
+
+```
+git clone https://github.com/opencv/dldt.git
+```
+
+If you explore the dldt directory, you'll see both the inference engine and the model optimizer. We are only concerned with the model optimizer. Navigating into the model optimizer folder you'll find several python scripts and text files. These are the scripts you call to run the model optimizer.
+
+
+### Install Prerequisites
+
+Install the Python packages required to run the model optimizer by running the script dldt/model-optimizer/install_prerequisites/install_prerequisites_tf.sh.
+```
+cd dldt/model-optimizer/install_prerequisites/
+./install_prerequisites_tf.sh
+cd ../../..
+```
+
+
+### Run the Model Optimizer 
+
+Running the model optimizer is as simple as calling the appropriate script, however there are many configuration options that are explained [here.](_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_Object_Detection_API_Models)
+
+```
+python dldt/model-optimizer/mo_tf.py \
+--input_model=ssd_inception_v2_coco_2018_01_28/frozen_inference_graph.pb \
+--tensorflow_use_custom_operations_config dldt/model-optimizer/extensions/front/tf/ssd_v2_support.json \
+--tensorflow_object_detection_api_pipeline_config ssd_inception_v2_coco_2018_01_28/pipeline.config \
+--reverse_input_channels
+```
+
+You should now see three files in your working directory, frozen_inference_graph.bin, frozen_inference_graph.mapping, and frozen_inference_graph.xml. These are your new models in the Intermediate Representation (IR) format and they are ready for use in the OpenVINO Inference Engine.
+
+_________________________________________________________________________________________________________________________________
+
+## Using the OpenVino Inference Engine
+
+Basic example on how to run the inference engine on your local machine 
+
+### Starting the Model Server
+
+The process is similar to how we start `Jupter notebooks` on our containers
+
+Run this command to spin up a OpenVino model fetched from GCP
+
+```bash
+docker run -p 8000:8000 stacks-tensorflow-mkl:latest bash -c ". /workspace/scripts/serve.sh && ie_serving model --model_name resnet --model_path gs://intelai_public_models/resnet_50_i8 --port 8000"
```

Comment:
update model path to :  gs://public-artifacts/intelai_public_models/resnet_50_i8